docs/source/configuration/environmentvariables.md

```diff
@@ -9,193 +9,192 @@ myst:
+    In the environment variable `ADDONS`, you can specify:
 
-#### Internationalization errors (i18n)
+    -   released (or published) packages that were installed previously in your environment and are already present in the `node_modules` directory,
+    -   or add-ons located in the `packages` folder of your project, such as Volto's testing packages.
 
-It will enable the log of missing i18n messages (in console).
+    `ADDONS` can be used to temporarily add an add-on to your build for testing purposes.
 
-## Use add-ons via the `ADDONS` environment variable
+    ```shell
+    yarn add volto-slate
+    ADDONS=volto-slate:asDefault yarn start
+    ```
 
-You can use the `ADDONS` environment variable to enable and configure add-ons in your project.
+    `ADDONS` can also be used to temporarily enable a feature or a set of customizations.
 
-When running your app, the add-ons will be loaded in the following order:
+    ```shell
+    # given a folder './packages/coresandbox', like in vanilla Volto
+    ADDONS=coresandbox:multilingualFixture yarn start
+    ```
 
-- the file `package.json`
-- programmatically set in the file `volto.config.js`
-- the environment variable `ADDONS`
+    You can specify multiple add-ons, seperated by commas:
+    
+    ```shell
+    ADDONS=test-addon,test-addon2 yarn start
+    ```
+    
+    You can specify profiles for installation:
+    
+    ```shell
+    ADDONS=test-addon:profile1,test-addon2:profile2 yarn start
+    ```
 
-In the environment variable `ADDONS`, you can specify:
+    The following code snippets demonstrate how to configure add-ons.
 
-- released (or published) packages that were installed previously in your environment and are already present in the `node_modules` directory,
-- or addons located in the `packages` folder of your project, such as Volto's testing packages.
+    First in `package.json`:
+    
+    ```json
+    "addons": [
+        "@kitconcept/volto-blocks-grid"
+    ]
+    ```
 
-`ADDONS` can be used to temporarily add an add-on to your build for testing purposes.
+    Then in `volto.config.js`:
 
-```bash
-yarn add volto-slate
-ADDONS=volto-slate:asDefault yarn start
-```
+    ```js
+    module.exports = {
+        addons: ['@eeacms/volto-accordion-block']
+    }
+    ```
 
-`ADDONS` can also be used to temporarily enable a feature or a set of customizations.
+    And finally using `ADDONS`:
 
-```bash
-# given a folder './packages/coresandbox', like in vanilla Volto
-ADDONS=coresandbox:multilingualFixture yarn start
-```
+    ```shell
+    yarn add volto-slate
+    ADDONS=volto-slate:asDefault yarn start
+    ```
 
-You can specify multiple add-ons, seperated by commas:
+    If you need to specify several add-ons, separate them with a semicolon (`;`):
 
-```bash
-ADDONS=test-addon,test-addon2 yarn start
-```
+    ```shell
+    yarn add volto-slate
+    ADDONS="volto-slate:asDefault;@kitconcept/volto-blocks-grid" yarn start
```

Comment:
@stevepiercy my only nitpick is about having this example here because it might be confusing to see that kitconcept/volto-blocks-grid is also referenced within the addons key while it is also in the package.json.
Given the resolution of package, volto.config, ADDONS key you might not be careful that you had it in package.json and think why you have this order.

Reply:
I focused only on reformatting existing content. I have no idea what I'm doing with Volto code. I'd appreciate any code improvement that you can provide.

I prefer not to add text to explain code. If code needs explanation, it may be a sign that it is not good code. But since the context is important to resolution, an explanation may be best. I'm fine with whatever you think is best.

Reply:
Thanks, your changes look good to me!